src/login/login.component.tsx

```diff
@@ -57,100 +59,104 @@ export default function Login(props: LoginProps) {
     border-radius: 4px;
   `;
 
-  return (
-    <div
-      css={css`
-        display: flex;
-        justify-content: center;
-        align-items: center;
-        width: 100vw;
-        height: 100vh;
-        background: #fafbfc;
-      `}
-    >
+  if (redirectToHomePage) {
+    return <Redirect to="/patient-dashboard" />;
+  } else {
+    return (
```

Comment:
This else branch is the same code as it was before, just indented. Github's diff is weird, but I didn't modify any of the JSX for the actual login form